ras_backstage/controllers/secure_messaging_controller.py

```diff
@@ -54,10 +49,11 @@ def update_label(encoded_jwt, message_id, label, action):
     response = request_handler('PUT', url, headers=headers, json=data)
 
     if not response or response.status_code != 200:
-        logger.error('Error updating label', status=response.status_code,
-                     message_id=message_id, label=label, action=action)
+        logger.error('Error updating label', message_id=message_id, label=label, action=action)
```

Comment:
Should we still log the status code here?

Reply:
It's logged out when we raise ApiError.

We can probably change this so that we don't need to log before raising the ApiError. We do this in frontstage-api already as it as has a slightly different ApiError https://github.com/ONSdigital/ras-frontstage-api/blob/master/frontstage_api/error_handlers.py#L27

It'd probably be good to introduce that in backstage too but not as part of this PR